cudax/samples/vector_add/vector.cuh

```diff
@@ -29,8 +29,7 @@
 #include <cuda/stream_ref>
 
 #include <cuda/experimental/__detail/utility.cuh>
-
-#include "param_kind.cuh"
+#include <cuda/experimental/__launch/param_kind.cuh>
```

Comment:
Suggestion: User should not rely on internal implementation details like sub headers

We should provide proper top level includes that collect all necessary features in e.g `<cuda/experimental/launch.cuh>`